indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskTuningConfig.java

```diff
@@ -281,7 +292,8 @@ public boolean equals(Object o)
       return false;
     }
     SeekableStreamIndexTaskTuningConfig that = (SeekableStreamIndexTaskTuningConfig) o;
-    return maxRowsInMemory == that.maxRowsInMemory &&
+    return Objects.equals(appendableIndexSpec, that.appendableIndexSpec) &&
+           maxRowsInMemory == that.maxRowsInMemory &&
```

Comment:
Could you please write an EqualsVerifier test for this. That should take care of the travis failures.

Reply:
@a2l007 Thanks for the tip. I wasn't aware of `EqualsVerifier`.
I added it and now it passes the tests but fails some integration tests that passed before.
I suspect there is an issue with the integration tests because sometimes the master branch also fails in these.